Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocTextValue.css

```diff
@@ -21,6 +21,7 @@ span.bocTextValue textarea
   display: inline-flex;
   align-items: center;
   width: fit-content;
+
```

Comment:
```suggestion
```

Reply:
Altrernatively, we could look into fixign it in here, not sure if that's best, though. I leave it up to you.